Subject: [PATCH] feat: i18n import zhCn
---
Index: packages/frontend/@n8n/i18n/src/index.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/packages/frontend/@n8n/i18n/src/index.ts b/packages/frontend/@n8n/i18n/src/index.ts
--- a/packages/frontend/@n8n/i18n/src/index.ts	(revision d31a6a9acc78099344a86a068f22c2fd7efdeec8)
+++ b/packages/frontend/@n8n/i18n/src/index.ts	(date 1758616182736)
@@ -4,6 +4,7 @@
 import { createI18n } from 'vue-i18n';
 
 import englishBaseText from './locales/en.json';
+import cnBaseText from './locales/zh-CN.json';
 import type { BaseTextKey, LocaleMessages, INodeTranslationHeaders } from './types';
 import {
 	deriveMiddleKey,
@@ -16,9 +17,9 @@
 
 export const i18nInstance = createI18n({
 	legacy: false,
-	locale: 'en',
+	locale: 'zh-CN',
 	fallbackLocale: 'en',
-	messages: { en: englishBaseText },
+	messages: { en: englishBaseText, 'zh-CN': cnBaseText, },
 	warnHtmlMessage: false,
 });
 
